Bio/NeuralNetwork/Gene/Schema.py

```diff
@@ -466,6 +466,13 @@ class SchemaFinder(object):
     """
     def __init__(self, num_schemas=100,
                  schema_finder=GeneticAlgorithmFinder()):
+        """Initialize the Schema Finder with its parameters.
+
+        Arguments:
+
+        o num_schemas -- the number of useful (positive fitness) schemas
```

Comment:
No letter ``o`` for bullet points - please use valid RST markup, which you can check using e.g. http://rst.ninjs.org/